commec/regulation/containers.py

```diff
@@ -0,0 +1,383 @@
+# Copyright (c) 2021-2024 International Biosecurity and Biosafety Initiative for Science
```

Comment:
I vaguely favour splitting this into a `dataclasses.py` and a `globals.py` if we can w/o circularity